docs/spec/relayer/Data-Requirements.md

```diff
@@ -162,3 +163,54 @@ The relayer connects to the node's CometBFT websocket interface and subscribes t
     - `EXISTS acknowledge_packet`
     - `EXISTS timeout_packet`
 
+## CometBFT gRPC
+
+The following are the gRPC endpoints that Hermes makes use of, listed in rough priority from most used to least used. 
+
+### `ibc.core.channel.v1.QueryClient`
+
+Queries for channel-associated data, such as packet commitments, unreceived packets, all channels associated with a given connection, etc.
+
+The requests that Hermes makes to this endpoint are:
+
+- QueryChannelClientStateRequest: Requests the client state associated with a specified channel.
+- QueryChannelsRequest: Requests all of the channels associated with the chain.
+- QueryConnectionChannelsRequest: Requests all of the channels associated with a specified connection.
+- QueryNextSequenceReceiveRequest: Requests the sequence number of the next receive packet for a specified channel.
+- QueryPacketCommitmentsRequest: Requests the packet commitments associated with a specified channel.
+- QueryPacketAcknowledgementsRequest: Requests the packet acknowledgments associated with a specified channel.
+- QueryUnreceivedAcksRequest: Requests the unreceived acknowledgments associated with a specified channel.
+- QueryUnreceivedPacketsRequest: Requests the unreceived packet sequences associated with a specified channel.
+
+> _*Note:*_ The `QueryPacketAcknowledgementsRequest`, `QueryUnreceivedAcksRequest`, and `QueryUnreceivedPacketsRequest` types each accept a Vector of `sequence`s in order to specify which packet commitments to fetch acknowledgements for. In the case where no `sequence`s are passed, the requests will simply return all acknowledgements for all outstanding packet commitments. 
+
+### `ibc.core.client.v1.QueryClient`
+
+Queries the client in order to fetch client and consensus states.
+
+The requests that Hermes makes to this endpoint are:
+
+- QueryClientStatesRequest: Requests all client states associated with the chain.
+- QueryConsensusStateHeightsRequest: Requests all the consensus state heights associated with a specified client.
+- QueryConsensusStatesRequest: Requests all the consensus states associated with a specified client.
+
+### `ibc.core.connection.v1.QueryClient`
+
+Queries the connection in order to fetch connection data and connected clients.
+
+The requests that Hermes makes to this endpoint are:
+
+- QueryClientConnectionsRequest: Requests all connections associated with a specified client.
+- QueryConnectionsRequest: Requests all connections associated with the chain.
+
+### `interchain_security.ccv.consumer.v1.QueryClient`
+
+Queries a CCV Consumer chain to fetch its staking parameters, notably its unbonding period and the number of historical entries that the chain keeps.
+
+### `cosmos.staking.v1beta1.QueryClient`
+
+Queries a Cosmos chain to fetch its staking parameters, most notably the chain's unbonding period.
+
+### `cosmos.base.node.v1beta1.ServiceClient`
```

Comment:
Can we list the specific queries we make using this client?

Reply:
[dfe0c6c](https://github.com/informalsystems/hermes/pull/3529/commits/dfe0c6cbe566553a4f1c85c73087e541ce22c97c)